src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/Internal/ConditionsList.tsx

```diff
@@ -106,30 +176,41 @@ const ConditionsList: React.FC<IConditionsListProps> = (props) => {
     // Update list everytime valueRanges and expressionType change
     useEffect(() => {
         setConditions(getConditionItems(valueRanges, expressionType));
-    }, [valueRanges, expressionType]);
-
-    // Callbacks
-    const handleOpenFlyout = useCallback(() => {
-        // TODO: Open callout
-        alert('New condition');
-    }, []);
+    }, [valueRanges, expressionType, getConditionItems]);
 
     return (
-        <div className={classNames.container}>
-            <Stack>
-                <CardboardList<Condition>
-                    listKey={LIST_KEY}
-                    items={conditions}
+        <>
+            <div className={classNames.container}>
```

Comment:
naming nit; we generally call the outer most container "root", following the pattern from Fluent.